src/redux/burgerMenu/burgerMenu.sagas.js

```diff
@@ -0,0 +1,42 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { push } from 'connected-react-router';
+import { setBurgerMenu, setBurgerMenuLoading } from './burgerMenu.actions';
+import { setError } from '../error/error.actions';
+import getItems from '../../utils/client';
+import { GET_BURGER_MENU } from './burgerMenu.types';
+
+export function* handleBurgerMenuLoad() {
+  yield put(setBurgerMenuLoading(true));
+  const query = `query {
```

Comment:
Move this to `burgerMenu.operations `folder

Reply:
Allready move